js/validate.js

```diff
@@ -0,0 +1,45 @@
+const form = document.querySelector('#my-form');
```

Comment:
i noticed when there is an error it clears all the inputs then state the error after it has cleared all inputs lets fix that

Reply:
Done